stdlib/smtplib.pyi

```diff
@@ -29,7 +29,7 @@ __all__ = [
 _Reply: TypeAlias = tuple[int, bytes]
 _SendErrs: TypeAlias = dict[str, _Reply]
 # Should match source_address for socket.create_connection
```

Comment:
This comment looks out of date — looks like the alias has diverged somewhat from https://github.com/python/typeshed/blob/9f0d0b9af7d4980431bbd62ca38497cf33308093/stdlib/_socket.pyi#L20

Reply:
We should just reuse that alias, not sure why I missed that. Source address just gets passed to `socket.create_connection`.